### PR TITLE
Refactoring code to create a common ansible-test-base

### PR DESCRIPTION
**Pull request by GomathiselviS**
writing a job to enable unit tests on collections.

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -747,12 +747,12 @@
         soft: true
     parent: ansible-network-openvswitch-appliance
     pre-run:
-      - playbooks/ansible-test-integration-base/pre.yaml
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:750]: all references to ansible-test-integration-base need to be changed

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -747,12 +747,12 @@
...
+      - playbooks/ansible-test-base/pre.yaml
       - playbooks/ansible-test-network-integration-base/pre.yaml
     run: playbooks/ansible-test-integration-base/run.yaml
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:752]: this needs to be moved too

File: playbooks/ansible-test-base/run.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (pabelanger) [inline playbooks/ansible-test-base/run.yaml:1]: make sure to delete the original file, as we are moving it

Comment (GomathiselviS) [reply playbooks/ansible-test-base/run.yaml:1]: I will remove the original file, once this runs successfully with openvswitch. The 'ansible-test-integration-base' is being used by other jobs.

File: zuul.d/jobs.yaml (hunk 2 above)
Comment (GomathiselviS) [reply zuul.d/jobs.yaml:752]: Done !

Comment (pabelanger) [reply zuul.d/jobs.yaml:752]: this should become
```
run: playbooks/ansible-test-base/run.yaml
```

File: zuul.d/jobs.yaml [hunk 4]
```diff
@@ -224,12 +224,12 @@
         soft: true
     parent: ansible-network-asa-appliance
     pre-run:
-      - playbooks/ansible-test-integration-base/pre.yaml
+      - playbooks/ansible-test-base/pre.yaml
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:227]: you'll also have to change zuul.d/ansible-cloud-jobs.yaml too

**Comment by pabelanger** [comment]
Can you remove WIP, and rebase against master, into a single commit?

Jobs looks to have passed.

**Pull request merged**